### ISSUE TITLE
Docs improvement/Quesion: How the ‘custom tag’ supposed to work?

### ISSUE THREAD

**Issue by jaandrle** [neoclide/coc-html#45]
Hi,
I’m trying to register custom elements tags. My naive approach was to create `coc-settings.json` in projects root folder with this content
```json
{
    "html.experimental.custom.tags": [
        "build/source/core/docs/globalComponents.json"
    ]
}
```
…so my root folder looks like:
```
build/
coc-settings.json
…
```

I have also tried to add this in `~/.vim/coc-settings.json` and different path formats (full path, …). But, nothing seems to work.

**tech. specs.:**
- os: ubuntu
- nodejs: v17.0.1
- [globalComponents.json.txt](https://github.com/neoclide/coc-html/files/7586967/globalComponents.json.txt): `.txt` is just because of github upload limitations. It is created by [web-component-analyzer - npm](https://www.npmjs.com/package/web-component-analyzer)

**Comment by chemzqm** [comment]
Seems your data is used for `html.customData.desc` which not supported by coc-html by now.

**Comment by chemzqm** [comment]
Fixed by b5f4aea, use `html.customData` configuration instead and the filepath should be relative to your workspace folder.
